packages/eslint-plugin/tests/rules/ban-ts-comment.test.ts

```diff
@@ -146,6 +177,19 @@ ruleTester.run('ts-ignore', rule, {
         '// @ts-ignore I think that I am exempted from any need to follow the rules!',
       options: [{ 'ts-ignore': 'allow-with-description' }],
     },
+    {
+      code: `
+/*
+// @ts-ignore running with long description in a block
```

Comment:
it would be great to have some tests that are not a combination of single line and multi-line comments.  If this works, it's not clear to me from the tests that it works because it's a multiline or because it's also a single line comment.

Reply:
Yea I was using the tests that were already present within our code just like that. Changed all multiline tests to multiline only.